components/style/matching.rs

```diff
@@ -588,46 +603,50 @@ trait PrivateMatchMethods: TElement {
         ChildCascadeRequirement::MustCascadeChildrenIfInheritResetStyle
     }
 
-    // FIXME(emilio, #20116): It's not clear to me that the name of this method
-    // represents anything of what it does.
-    //
-    // Also, this function gets the old style, for some reason I don't really
-    // get, but the functions called (mainly update_style_for_animation) expects
-    // the new style, wtf?
     #[cfg(feature = "servo")]
-    fn update_animations_for_cascade(
+    fn collect_and_update_style_for_expired_transitions(
         &self,
         context: &SharedStyleContext,
+        node: OpaqueNode,
```

Comment:
`node` is always `self.as_node().opaque()`. Either assert it, or make this a static function as it doesn't use `self`?

Reply:
Thanks for the review! I've converted these two to static methods, just for code organization purposes.